.github/labeler.yml

```diff
@@ -0,0 +1,15 @@
+core:
```

Comment:
Should probably add `util` to this, for the time being.

Reply:
yeah, stuff like build scripts live in the folder so makes sense.